src/components/widget/RawWidget.js

```diff
@@ -129,9 +130,16 @@ export class RawWidget extends Component {
   // Datepicker is checking the cached value in datepicker component itself
   // and send a patch request only if date is changed
   handlePatch = (property, value, id, valueTo, isForce) => {
-    const { handlePatch } = this.props;
+    const { handlePatch, widgetData } = this.props;
     const willPatch = this.willPatch(property, value, valueTo);
 
+    const fieldData = widgetData.find(widget => widget.field === property);
+    let isCostPriceInput = false;
+
+    if (fieldData && fieldData.widgetType === 'CostPrice') {
```

Comment:
do we still need this? AFAIU your PR is about a generic solution..?

Reply:
I don't want to go so far and turn this for all fields. It's gonna be madness, I'm pretty sure.